recipes/jupyterlite-core/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "jupyterlite-core" %}
```

Comment:
as noted on https://github.com/jupyterlite/jupyterlite/issues/383:

if `jupyterlite` and `jupyterlite-core` will always be released in lockstep, keeping them together as a multi-output is going to be useful.